src/mmw/js/src/modeling/controls.js

```diff
@@ -318,38 +318,26 @@ var ModificationsView = ControlView.extend({
         modContentRegion: '.mod-content-region'
     },
 
-    toggleDropdown: function() {
-        if (this.model.get('dropdownOpen')) {
-            this.openDropdown();
+    toggleDropdown: function(model, dropdownOpen) {
+        var dropdown = this.$(this.ui.dropdown);
```

Comment:
Is the wrapping `this.$()` necessary? I thought `this.ui.dropdown` had already selected it; `this.ui.dropdown.addClass('open')` works just as well.

Reply:
I was wondering about that. Thanks for checking, will switch.